cmd/rollkit/commands/intercept.go

```diff
@@ -79,22 +101,9 @@ func RunRollupEntrypoint(rollkitConfig *rollconf.TomlConfig, args []string) erro
 		entrypointSourceFile = rollkitConfig.Entrypoint
 	}
 
-	// The entrypoint binary file is always in the same directory as the rollkit.toml file.
-	entrypointBinaryFile := filepath.Join(rollkitConfig.RootDir, rollupBinEntrypoint)
-
-	if !cometos.FileExists(entrypointBinaryFile) {
-		if !cometos.FileExists(entrypointSourceFile) {
-			return fmt.Errorf("no entrypoint file: %s", entrypointSourceFile)
-		}
-
-		// try to build the entrypoint as a go binary
-		buildArgs := []string{"build", "-o", entrypointBinaryFile, entrypointSourceFile}
-		buildCmd := exec.Command("go", buildArgs...) //nolint:gosec
-		buildCmd.Stdout = os.Stdout
-		buildCmd.Stderr = os.Stderr
-		if err := buildCmd.Run(); err != nil {
-			return fmt.Errorf("failed to build entrypoint: %w", err)
-		}
+	entrypointBinary, err := buildEntrypoint(rollkitConfig.RootDir, entrypointSourceFile, false)
```

Comment:
same applies here, it isn't a binary, it is a filepath to the binary that was built.